sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/SchemaRegistry.java

```diff
@@ -55,8 +64,12 @@
     }
   }
 
-  private final Map<TypeDescriptor, SchemaEntry> entries = Maps.newHashMap();
-  private final List<SchemaProvider> providers = Lists.newArrayList();
+  Map<TypeDescriptor, SchemaEntry> entries = Maps.newHashMap();
```

Comment:
It doesn't look like anything requires the `private final` to be removed?

Reply:
fixed